save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ContestView.kt

```diff
@@ -102,23 +139,16 @@ class ContestView : AbstractView<ContestViewProps, ContestViewState>(false) {
             nav {
                 className = ClassName("nav nav-tabs mb-4")
                 ContestMenuBar.values()
-                    .forEachIndexed { i, contestMenu ->
-                        li {
-                            className = ClassName("nav-item")
-                            val classVal =
-                                    if ((i == 0 && state.selectedMenu == null) || state.selectedMenu == contestMenu) {
-                                        " active font-weight-bold"
-                                    } else {
-                                        ""
-                                    }
-                            p {
-                                className = ClassName("nav-link $classVal text-gray-800")
-                                onClick = {
-                                    if (state.selectedMenu != contestMenu) {
-                                        setState {
-                                            selectedMenu = contestMenu
-                                        }
-                                    }
+                  .forEachIndexed { i, contestMenu ->
+                      li {
+                          className = ClassName("nav-item")
+                          val classVal = if ((i == 0 && state.selectedMenu == null) || state.selectedMenu == contestMenu) " active font-weight-bold" else ""
```

Comment:
Why did you remove `(i == 0 && state.selectedMenu == null)` from `OrganizationView` and `AwesomeBenchmarksView`, but left it here?

Reply:
Returned this in start version